README.md

```diff
@@ -164,12 +164,13 @@ The library is compatible with _AoT compilation_.
     * `window`, `document`, `navigator` and other browser types do not exist on the server
     * don't manipulate the _nativeElement_ directly
 
-## Built with this starter
+## Angular 2 autocomplete component  this starter
```

Comment:
Hi @oferh, could you redo the title of the paragraph? so I immediately do the merge. Thank you!